packages/eslint-config-typescript/README.md

```diff
@@ -0,0 +1,36 @@
+# Eslint
+
+Provide reusable typescript and react eslint configs
+
+1. Install the peerDependencies of eslint configs:
+
+```sh
+npx install-peerdeps -Y --dev @offchainlabs/eslint-config-typescript/base
+npx install-peerdeps -Y --dev @offchainlabs/eslint-config-typescript/react
+```
+
+2. Usage
+
+```ts
```

Comment:
nit: same thing, which file should this go into

Reply:
Update with the different possibilities